index.html

```diff
@@ -977,22 +986,20 @@
         </li>
       </ol>
       <p>
-        The <dfn>serializer</dfn> for a <a>PushSubscription</a> invokes the following steps:
+        The <dfn>toJSON</dfn> method when invoked MUST run the following steps:
       </p>
       <ol>
-        <li>Let <var>map</var> be an empty map.
+        <li>Let <var>json</var> be a new <code>PushSubscriptionJSON</code> dictionary.
         </li>
-        <li>Add an entry to <var>map</var> whose key name is <code>endpoint</code> and whose value
-        is the result of converting the <code>endpoint</code> attribute to a serialized value. The
+        <li>Set the <code>endpoint</code> property of <var>json</var> to the result of converting
+        the <a data-link-for="PushSubscription">endpoint</a> attribute to a serialized value. The
         <a>user agent</a> MUST use a serialization method that does not contain input-dependent
         branchs (that is, one that is constant time). Note that a URL - as ASCII text - will not
         ordinarily require special treatment.
         </li>
-        <li>Add an entry to <var>map</var> whose key name is <code>expirationTime</code> and whose
-        value is the result of converting the <code>expirationTime</code> attribute to a serialized
-        value.
-        </li>
-        <li>Let <var>keys</var> be an empty map.
+        <li>Set the <code>expirationTime</code> property of <var>json</var> to the result of
+        converting the <a data-link-for="PushSubscription">expirationTime</a> attribute to a
```

Comment:
Done. Also for `endpoint`. I like that `get the underlying value` avoids duplication.